napari/_qt/containers/qt_layer_model.py

```diff
@@ -20,7 +20,7 @@ def data(self, index: QModelIndex, role: Qt.ItemDataRole):
         if role == Qt.EditRole:  # used to populate line edit when editing
             return layer.name
         if role == Qt.ToolTipRole:  # for tooltip
-            return layer.name
+            return f'name: {layer.name},  plugin: {layer.source.reader_plugin}'
```

Comment:
We should probably add the `trans._` calls to make this localizable.

See: https://napari.org/developers/contributing.html#translations